fleetctl/stop.go

```diff
@@ -85,6 +85,11 @@ func runStopUnit(args []string) (exit int) {
 	}
 
 	exit = tryWaitForUnitStates(stopping, "stop", job.JobStateLoaded, getBlockAttempts(), os.Stdout)
+	if exit == 0 {
+		stderr("Successfully stopped units %v.", stopping)
+	} else {
+		stderr("Failed to stop units %v. exit == %d.", stopping, exit)
```

Comment:
My apologies for missing this previously. Why do we print the return value here? We already exit the process with that value so it can be easily inspected. Could you remove that from the string, please?